src/jobflow_remote/jobs/daemon.py

```diff
@@ -1134,25 +1151,31 @@ def _check_running_runner(
             "user",
             "daemon_dir",
         ]
-        for data in data_to_check:
-            if local_data[data] != db_data[data]:
-                break
-        else:
+        diffs = [d for d in data_to_check if local_data[d] != db_data[d]]
+
+        if not diffs:
             logger.info(
                 "The DB reports that there is a running runner and it corresponds to this machine"
             )
             return None
 
-        error = (
-            "A daemon runner process may be running on a different machine.\n"
-            "Here is the information retrieved from the database:\n"
-            f"- hostname: {db_data['hostname']}\n"
-            f"- project_name: {db_data['project_name']}\n"
-            f"- start_time: {db_data['start_time']}\n"
-            f"- last_pinged: {db_data['last_pinged']}\n"
-            f"- daemon_dir: {db_data['daemon_dir']}\n"
-            f"- user: {db_data['user']}"
+        error = "A daemon runner process associated to this database may be already running.\n"
```

Comment:
Maybe to this project ?
```suggestion
        error = "A daemon runner process associated to this project may be already running.\n"
```

Reply:
Here I picked the word specifically, since if you created two projects with the same DB (as a user likely did a few days ago), it will be wrong stating that the runner is associated with the same "project". Still I agree that the "database" word may be confusing. Do you have other suggestions?

Reply:
No you are right let's keep it this way. Or maybe refer one more time to the documentation where we insist that projects should not share the same collection/database.